README.md

```diff
@@ -41,12 +41,18 @@ To submit a new RFC, follow these steps:
 1. Copy the appropriate template file from the `templates` directory into the appropriate `designs` subdirectory (such as `designs/2018-typescript-support/README.md`). Be sure to name your file `README.md` so it is easily viewable in the GitHub interface.
 1. If you want to include images in your RFC, place them in the same directory as the `README.md`.
 1. Fill in the RFC. Please fill in every section in the template with as much detail as possible.
-1. Submit a pull request to this repo with all of your files.
-1. You will receive feedback both from the ESLint community and from the ESLint team. You should be prepared to update your RFC based on this feedback. The goal is to build consensus on the best way to implement the suggested change.
-1. When all feedback has been incorporated, the ESLint TSC will determine whether or not to accept the RFC.
+1. Submit a pull request to this repo with all of your files. This begins the approval process (detailed below).
 1. RFCs that are accepted will be merged directly into this repo; RFCs that are not accepted will have their pull requests closed without merging.
 
-## The RFC life-cycle
+## The RFC Approval Process
+
+When an RFC is submitted, it goes through the following process:
+
+1. **Initial commenting period (21 days)** - the community and ESLint team are invited to provide feedback on the proposal. During this period, you should expect to update your RFC based on the feedback provided. Very few RFCs are ready for approval without edits, so this period is important for fine-tuning ideas and build consensus. (A PR in the initial commenting period has the **Initial Commenting** label applied.) The initial commenting period may be extended with a TSC vote if the RFC has been unable to reach consensus but the team believes it worthwhile to continue developing the RFC. The TSC may also decide to abandon an RFC that has failed to reach consensus after 21 days.
+1. **Final commenting period (7 days)** - when all feedback has been addressed, the pull request enters the final commenting period where ESLint TSC members provide their final feedback and either approve of the pull request or state their disagreement. (A PR in the final commenting period has the **Final Commenting** label applied.) ESLint TSC members are notified through GitHub when an RFC has passed into the final commenting period.
```

Comment:
I agree. I think allowing the PR author to request the final commenting period makes sense. My thoughts:

1. I think we can trust TSC members to apply the "Final Commenting" label when they believe it's time.
1. For non-TSC members, it seems like it should be good enough to have one TSC member review the RFC to ensure it's ready and apply the label.

(Trying to avoid a vote for every part of the process.)